layers/+misc/multiple-cursors/README.org

```diff
@@ -60,3 +59,43 @@ For easy navigation you also have the following:
 | ~C-t~       | evil-mc-skip-and-goto-next-match   |
 | ~C-M-j~     | evil-mc-make-cursor-move-next-line |
 | ~C-M-k~     | evil-mc-make-cursor-move-prev-line |
+|             |                                    |
+** =multiple-cursors=
+The =multiple-cursors= backend provides the following key bindings to
+insert new cursors:
+
+| Key binding | Description                       |
+|-------------+-----------------------------------|
+| ~g r r~     | mc/mark-more-like-this-extended   |
+| ~g r a~     | mc/mark-all-like-this             |
+| ~g r d~     | mc/mark-all-like-this-in-defun    |
+| ~g r l~     | mc/edit-lines                     |
+
+These special bindings manipulate text under cursors:
+
+| Key binding | Description                   |
+|-------------+-------------------------------|
+| ~g r s s~   | mc/sort-regions               |
+| ~g r s r~   | set-rectangular-region-anchor |
+| ~g r s m~   | mc/mark-sgml-tag-pair         |
+| ~g r s n~   | mc/insert-numbers             |
+| ~g r s l~   | mc/insert-letters             |
+| ~g r s t~   | mc/reverse-regions            |
+* Notes
```

Comment:
Add a newline to leave a space between the key binding table and the following header.

Reply:
Done